### ISSUE TITLE
`create_function()` depreciated in PHP 7.2

### ISSUE THREAD

**Issue by wottpal** [punkave/phpQuery#15]
Just for reference I've created a fork of this repository where I've replaced all occurrences of `create_function()` with anonymous functions. 

~~https://github.com/wottpal/phpQuery~~
UPDATE: Use @slaith's repo https://github.com/slaith/phpQuery when https://github.com/slaith/phpQuery/pull/3 is merged

Dennis

**Comment by ahfeel** [comment]
Hey @wottpal, seems this repo is totally abandoned.. I'm having these warnings too, and I see you had the good idea to merge @slaith serialization fixes on your branch too, that's great. Would you mind publishing it on packagist so we can use it with composer easily ? Also it may be good to update it based on @slaith latest commits :)

**Comment by slaith** [comment]
Hi,
Thanks for pointing out @ahfeel I created the package on packageist. https://packagist.org/packages/somesh/php-query

You can create a pull request if you like. I'll merge in the changes and publish it on packagist again. @	wottpal

**Comment by ahfeel** [comment]
Oh great you're jumpin' in @slaith ! I'm already using your package so merging these changes in your repo would be awesome.
Great stuff community ! :)

**Comment by slaith** [comment]
Yes, I do keep an eye on this package. Good to know its helping you :)

**Comment by wottpal** [comment]
Hey I didn't even think someone notices my fork 😅

I did a pull request to @slaith's repo here https://github.com/slaith/phpQuery/pull/3

**Comment by slaith** [comment]
I've merged it, though it should break before PHP 5.3

**Comment by DrLightman** [comment]
Hello,

https://packagist.org/packages/somesh/php-query

packagist seems broken:

```
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Downloading somesh/php-query (2.0)
    Failed to download somesh/php-query from dist: The "https://api.github.com/repos/slaith/phpQuery/zipball/91519876bb56ddc453a05be05d61c1a9489864c8" file could not be downloaded (HTTP/2 404 )
```
